sky/compositor/picture_layer.cc

```diff
@@ -3,11 +3,19 @@
 // found in the LICENSE file.
 
 #include "sky/compositor/picture_layer.h"
+
 #include "base/logging.h"
+#include "sky/compositor/checkerboard.h"
+#include "sky/compositor/raster_cache.h"
+
+#define ENABLE_RASTER_CACHE 0
 
 namespace sky {
 namespace compositor {
 
+// TODO(abarth): Make this configurable by developers.
+const bool kDebugCheckerboardRasterizedLayers = false;
```

Comment:
You can add an entry in compositor_options. It already has hooks in Dart so developers will be able to toggle it at will.